runtime/ts/storage/firebase-storage.ts

```diff
@@ -358,7 +358,7 @@ class FirebaseVariable extends FirebaseStorageProvider {
     assert(data.version >= this.version);
     if (this.version != version) {
       // A new local modification happened while we were writing the previous one.
-      return this._persistChangesImpl();
+      this._persistChangesImpl();
```

Comment:
I think this still needs to return, to capture the promise chain; but it's just passing on Promise<void> (we await this.persisting at one point, which is assigned the output of this._persistChangesImpl).

Reply:
agreed, reverted in 41f09b4